pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/PulsarAdmin.java

```diff
@@ -147,11 +146,6 @@ static PulsarAdminBuilder builder() {
      */
     String getServiceUrl();
 
-    /**
-     * @return the client Configuration Data that is being used
-     */
-    ClientConfigurationData getClientConfigData();
```

Comment:
Is this a breaking change? Or was this method introduced on master branch?

Reply:
@eolivelli  thanks for the reivew. `ClientConfigurationData` is located in `pulsar-client-original`, which should not be as a dependency for `pulsar-client-admin-api`. So I have kept the impl of `getClientConfigData()` in `PulsarAdminImpl`.

Reply:
I believe the this method is pretty useless so I am fine to remove it.
We are also making this change as part of a new major release, so this "minor" API breaking change is acceptable from my point of view.

I just wanted to double check